Examples/Swift/AppDelegate+CarPlay.swift

```diff
@@ -47,19 +47,42 @@ extension AppDelegate: CarPlayManagerDelegate {
         NavigationViewController.carPlayManagerDidEndNavigation(carPlayManager, window: window)
     }
     
+    func favoritesListTemplate() -> CPListTemplate {
+        let mapboxSFItem = CPListItem(text: FavoritesList.POI.mapboxSF.rawValue,
```

Comment:
For future reference, a more realistic example probably wouldn’t involve a formal symbol for each POI. Instead, there would be an array of POIs.